### ISSUE TITLE
Bower needs to specify angular as a dependency for algoliasearch.angular.js

### ISSUE THREAD

**Issue by dderiso** [algolia/algoliasearch-client-js#182]
Thanks for the great lib!

```
<script src="../bower_components/algoliasearch/dist/algoliasearch.angular.js"></script>
<script src="../bower_components/angular/angular.js"></script>
```

should be
```
<script src="../bower_components/angular/angular.js"></script>
<script src="../bower_components/algoliasearch/dist/algoliasearch.angular.js"></script>
 ```

**Comment by vvo** [comment]
Thanks! I cannot find the referenced code snippets of yours. Can you point me to where you saw them?

Thanks

**Comment by esquevin** [comment]
What you can do is override the configuration in your own `bower.json` file

```json
    "overrides": {
        "algoliasearch": {
            "main": [
                "dist/algoliasearch.angular.js"
            ],
            "dependencies": {
                "angular": "1.3"
            }
        }
    }
```

**Comment by vvo** [comment]
@dderiso Can you precise your issue and reopen if needed, thanks

**Comment by esquevin** [comment]
I encountered a similar issue. When using the angular version of algoliasearch-client-js, there is an implied dependency on angular, when trying to automatically inject your js dependencies in your project, algoliasearch-client-js is injected before angular, and it breaks the build.
I use `angularFilesort` to automatically import my dependencies in the proper order. But angular cannot be listed as a dependency in algoliasearch-client-js bower.json which serves all versions of the library.

That's why I had to override algoliasearch-client-js dependencies in my own bower.json file
